Makefile.am

```diff
@@ -239,6 +239,7 @@ if HAVE_CMOCKA
         test-find-uid \
         test-io \
         test-negcache \
+        negcache_2-tests \
```

Comment:
Is there something special about the second test that it can't be merged? If it can't be, maybe some better name than negcache_2 would be nice.

Reply:
Addressed in a separate ticket https://pagure.io/SSSD/sssd/issue/3989